File: assignment 10 - C# practice problems/21.cs
//Write a c# function to add specified minutes to a Date object.

using System;
namespace MyNameSpace
{
    class Program21
    {
        public static void AddMinutesToDateObjectDriverFunction()
        {


            Console.WriteLine("Enter 2 dates in dd/mm/yyyy format");
            var dateInput1 = Console.ReadLine();
            int mintesToBeAdded = int.Parse(Console.ReadLine());
            var split1 = dateInput1.Split('/');

            var date1Obj1 = new DateTime(int.Parse(split1[2]), int.Parse(split1[1]), int.Parse(split1[0]), 12, 0, 0);

            Console.WriteLine("Date Before Adding: " + date1Obj1.AddMinutes(0));
            date1Obj1.AddMinutes(mintesToBeAdded);
            Console.WriteLine("Date after Adding: " + date1Obj1.AddMinutes(mintesToBeAdded));



        }
    }
}

File: assignment 3 - js practice problems/web-dev-starter/21.js
//Write a JavaScript function to add specified minutes to a Date object.
function addMinutesToDateObject(minutesToBeAdded) {

    let date = new Date(1998, 3, 19);

    console.log("time before adding " + minutesToBeAdded +
        " minutes " + date.toLocaleTimeString());

    const MILLIS_IN_MINUTE = 60000
    let timeInMillis = date.getTime() + minutesToBeAdded * MILLIS_IN_MINUTE;
    date = new Date(timeInMillis);

    console.log("time after adding " + minutesToBeAdded +
        " minutes " + date.toLocaleTimeString());

}

function addMinutesToDateObjectDriverFunction() {
    let minutesInput = prompt("Enter minutes to be added");

    addMinutesToDateObject(minutesInput);
}

addMinutesToDateObjectDriverFunction();